src/config.ts

```diff
@@ -59,6 +63,10 @@ export const CONFIG = {
         row: 10,
         col: 2,
       },
+      keyword: {
```

Comment:
Is this really necessary? The way we parse the response from the LLM should be "our" thing. I don't see us exposing the "template" keywords for titles, etc., so I would get rid of this (and the associated user-level settings in the frontend, too).

Reply:
Since it's not visible to the user (and we assume the config sheet structure is fixed and not really user-tuneable) AND this is used solely for few-shot description examples handling, we should keep it.

Reply:
But it IS visible to end-users -> https://screenshot.googleplex.com/6XYoqtiFaSQS5vN
Or am I missing something?

Reply:
I see the other comment now. Marking this as resolved.